alti/lib/raster/georaster.py

```diff
@@ -98,9 +98,9 @@ def get_height_for_coordinate(self, x, y):
                 format_to_unpack = "<h"
             else:
                 format_to_unpack = "<l"
-        return unpack(format_to_unpack, self.file.read(self.data_size))[0]
-        # we let the file opened, so that we can read further point more efficiently
-        # (the file will be closed by a call to close_file() or by the __del__ function)
+        value = unpack(format_to_unpack, self.file.read(self.data_size))[0]
+        self.close_file()
```

Comment:
Using pythons `with ...` context manager accessing resources handles closing automatically

Reply:
You may also use `try..except..finally`